enclave-runtime/src/lib.rs

```diff
@@ -589,6 +722,9 @@ fn get_stf_state(
 ///
 /// For fairness, the [`max_exec_duration`] is split equally among all shards. Leftover time from
 /// the previous shard is evenly distributed to all remaining shards.
+///
+/// Todo: This will probably be used again if we decide to make sidechain optional?
+#[allow(unused)]
 fn exec_tops_for_all_shards<PB, SB, O, R>(
```

Comment:
I might be lacking understanding of the side-chain feature here: Are TOPs not executed with the side-chain feature at all? Or just in a different fashion?

Reply:
I have seen your other comment regarding this, but will still comment here for completeness.

Only the function that executes tops for a single shard is currently used.